relay-monitors/src/lib.rs

```diff
@@ -257,36 +237,11 @@ mod tests {
     fn process_empty_slug() {
         let json = r#"{
           "check_in_id": "a460c25ff2554577b920fcfacae4e5eb",
-          "monitor_slug": "🚀🚀🚀",
+          "monitor_slug": "",
           "status": "in_progress"
         }"#;
 
         let result = process_check_in(json.as_bytes());
         assert!(matches!(result, Err(ProcessCheckInError::EmptySlug)));
     }
-
-    #[test]
-    fn slugify_empty_string() {
-        assert_eq!("", slugify(""));
-    }
-
-    #[test]
-    fn slugify_truncate_honors_trim() {
-        let input = "-".repeat(SLUG_LENGTH + 10) + "hello";
-        assert_eq!("hello", slugify(&input));
-    }
-
-    #[test]
-    fn slugify_trim_at_truncate() {
-        let expected = "a".repeat(SLUG_LENGTH - 1);
-        let input = expected.clone() + "-stripped";
-        assert_eq!(expected, slugify(&input));
-    }
-
-    #[test]
-    fn slugify_unicode() {
-        let input = "🚀🚀🚀\tmyComplicated_slug\u{200A}name is here...";
-        let expected = "mycomplicated_slug-name-is-here";
-        assert_eq!(expected, slugify(input));
-    }
```

Comment:
@evanpurkhiser Can we keep these tests / add some more to make sure the cropping works as expected? You could put lines 136-138 in a helper function to make it unit testable.

Reply:
Ah yeah I'll bring this back!